_includes/azure-locations.md

```diff
@@ -0,0 +1,29 @@
+|  Location | SQL Statement |
+|  eastasia (East Asia) | `INSERT into system.locations VALUES ('region', 'eastasia', 22.267, 114.188)` |
```

Comment:
The table is broken without a separation between the header row and the first normal row:

```
|  Location | SQL Statement |
|----------|-----------------|
|  eastasia (East Asia) | `INSERT into system.locations VALUES ('region', 'eastasia', 22.267, 114.188)` |
```

Can you put that in place please?

Reply:
Great catch! I'm sorry for missing that. Done.

Reply:
Thanks, Alex! This PR LGTM now. @Amruta-Ranade, once you're back, please add the GCE coordinates as well. I've assign this issue to you: https://github.com/cockroachdb/docs/issues/2885.